k8s/prod/common/divorce/div-cos.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: div-cos
+  namespace: divorce
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: div-cos
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: div-cos
+    version: 1.3.0
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "baaed5d0-7666-4dce-bf74-edfceeb566f9"
+      image: hmctspublic.azurecr.io/div/cos:prod-aed712be
+      environment:
```

Comment:
Please compare the below with https://github.com/hmcts/div-case-orchestration-service/blob/master/infrastructure/variables.tf and https://github.com/hmcts/div-case-orchestration-service/blob/master/infrastructure/prod.tfvars